.github/workflows/desktop-build.yml

```diff
@@ -236,26 +236,37 @@ jobs:
             xcode: "14.3.1"
             type: Release
             make_package: 1
+            qt_version: 6.6.*
```

Comment:
it feels like there should be a way to reduce the duplication of all these params across the matrix's entries, but I have to confess I got a bit lazy to do it in this pr. IMO it's worth it to create a ticket to be done later

Reply:
done using env vars